specs/Deployment/DeploymentAPI.md

```diff
@@ -3,101 +3,117 @@
 
 The API addresses #1 by a simple query of the version needed and a check for OS updates.
 
-#2 has two possible answers.
+#2 has the following two possible answers.
 
--   First, the developer can install these packages directly through the app, if possible. These
-    MSIX packages can be acquired through
+-   The developer can install these packages directly through the app, if possible. These MSIX
+    packages can be acquired through
     [Downloads page](https://docs.microsoft.com/windows/apps/windows-app-sdk/downloads).
 -   Alternatively, unpackaged apps that are full trust or MSIX packaged apps that have the
     packageManagement restricted capability can use the Deployment API to get these packages
     installed.
 
+#3 has the following two possible answers.
+
+-   The developer can repair these packages directly through the app by reinstalling them, if
+    possible. These MSIX packages can be acquired through
+    [Downloads page](https://docs.microsoft.com/windows/apps/windows-app-sdk/downloads).
+-   Alternatively, unpackaged apps that are full trust or MSIX packaged apps that have the
+    packageManagement restricted capability can use the Deployment API to get these packages repair.
+
 # Examples
 
 ## GetStatus
 
-This is the means by which an app or program can call to verify that all required WinAppSDK packages
-are present to a minimum version needed by the framework loaded with the app. A process should call
-GetStatus() after process initialization and before using Windows App Runtime content (e.g. in
-main() or WinMain()).
+This is the API an app or program can call to verify that all required WinAppSDK packages are
+installed to a minimum version needed by the Framework loaded with the app and that the packages are
+in good state. A process should call GetStatus() after process initialization and before using
+Windows App Runtime content (e.g. in Main() or WinMain()).
 
 Developers call this method without parameters, as all information about the version, channel, and
-architecture are derived from the current framework package loaded.
+architecture are derived from the current Framework package loaded.
 
 ```C#
     DeploymentResult result = DeploymentManager.GetStatus();
     if (result.Status == DeploymentStatus.Ok)
     {
-        // WindowsAppRuntime is in a good state and ready for use.
+        // WindowsAppRuntime is installed and is in a good state and ready for use.
     }
     else
     {
-        // WindowsAppRuntime is not in a good state.
-        // Should call Initialize on another thread, or not use features that require the
-        // WindowsAppRuntime. result.ExtendedError has the error code that has the reason
-        // for the Status.
+        // WindowsAppRuntime is either not installed or not in a good state (can't be both at same time).
+        if (result.Status == DeploymentStatus.PackageInstallRequired)
```

Comment:
What happens if an app developer upgrades to WinAppSDK 1.3 but forgets to update this part of their code? In other words, what happens if the app just calls Initialize for all non-OK deployment statuses instead of calling Repair sometimes?

I hope the answer is "Everything still works, though maybe less efficiently than if you had used Repair."